package.json

```diff
@@ -43,6 +43,7 @@
     "@types/qs": "^6.9.7",
     "ajv": "^6.12.6",
     "async": "^3.2.0",
+    "axios": "^1.2.5",
```

Comment:
I think you should use [got](https://www.npmjs.com/package/got) library for making API calls, as it is already being used in a different PR.